lib/interfaces/mongoose-options.interface.ts

```diff
@@ -8,7 +8,16 @@ export interface MongooseModuleOptions extends ConnectOptions {
   connectionName?: string;
   connectionFactory?: (connection: any, name: string) => any;
   connectionErrorFactory?: (error: MongooseError) => MongooseError;
+  /**
+   * @deprecated Method has been moved to `factoryOptions`
+   */
   lazyConnection?: boolean;
+  factoryOptions?: MongooseFactoryOptions;
```

Comment:
I just meant to update the method signature, the module's options object could stay as is 😅

Reply:
My bad, I'll make the updates ASAP.

Reply:
@kamilmysliwiec Sry for the confusion/misunderstanding. I've made the changes as suggested.